spouts/rss/feed.php

```diff
@@ -243,7 +243,7 @@ public function getContent() {
      * @return string icon url
      */
     public function getIcon() {
-        if (isset($this->faviconUrl)) {
+        if ($this->faviconUrl !== '') {
```

Comment:
Then I would prefer that. I would like to have as much type checking as possible in PHP.